src/test/regress/expected/multi_schema_support.out

```diff
@@ -1084,3 +1084,46 @@ SET search_path TO public;
 ALTER TABLE test_schema_support.nation_hash SET SCHEMA public;
 WARNING:  not propagating ALTER ... SET SCHEMA commands to worker nodes
 HINT:  Connect to worker nodes directly to manually change schemas of affected objects.
+-- test schema propagation with user other than current user
+CREATE USER "test-user";
```

Comment:
> run_command_on_master_and_workers

And we prefer `coordinator` over `master`  😄  `run_command_on_coordinator_and_workers()` should be better